common/lib/common-definitions/src/logger.ts

```diff
@@ -120,3 +120,23 @@ export interface ITelemetryLogger extends ITelemetryBaseLogger {
      */
     shipAssert(condition: boolean, event?: ITelemetryErrorEvent): void;
 }
+
+// ////////////////////// fluidError.ts in common-definitions /////////////////////////////
+/**
+ * Can be decl merged to add stuff like packagename, odspErrorResponse, etc
+ * The logger impl can choose to log anything in here as appropriate, but by default it's not
+ */
+export interface ISensitiveDebugData {
+    innerError?: any;
+}
+
+export interface IFluidError {
+    errorType: string;
```

Comment:
what's the advantage of making this a top level property?

Reply:
So classes that extend `abstract FluidError` can provide it statically.